testdata/xml-validate/data/content_violation.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
These were all the violations I was able to come up with, that passed the xml schema validation. If you can think of more, please comment on it.